charts/threat-protection/values.yaml

```diff
@@ -0,0 +1,119 @@
+threat_detection: 
+  aktoApiSecurityDetection:
+    env:
+      aktoAccountName: Helios
+      aktoConfigName: staging
+      aktoInstanceType: RUNTIME
+      aktoKafkaBrokerMal: localhost:29092
+      aktoKafkaBrokerUrl: 127.0.0.1:29092
+      aktoKafkaGroupIdConfig: asdf
+      aktoKafkaMaxPollRecordsConfig: "100"
+      aktoKafkaTopicName: akto.api.logs
+      aktoTrafficBatchSize: "100"
+      aktoTrafficBatchTimeSecs: "10"
+      puppeteerReplayServiceUrl: http://akto-puppeteer-replay:3000
+      aktoKafkaProtectionUrl: localhost:29092
+      requestProtectionEnabled: "true"
+    enabled: true
+    image:
+      repository: public.ecr.aws/aktosecurity/mirror-api-logging
```

Comment:
for now, use a runtime image to test the helm chart. It closely resembles the threat-detection chart.